source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -0,0 +1,201 @@
+====================
+MongoClient Behavior
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+Overview
+--------
+
+In this guide, you can learn about the different settings to control
+the behavior of your ``MongoClient``.
+
+You can control the behavior of your ``MongoCleint`` by creating and passing in a 
+`MongoClientSettings <{+api+}/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.html>`__
+object to the `MongoClients.create() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoClients.html#create(com.mongodb.MongoClientSettings)>`__
+method.
+
+To create a ``MongoClientSettings`` object, use the
+``MongoClientSettings.builder()`` method and chain methods to specify
+your desired settings. After chaining all the settings you want, use the
+``build()`` method to create the ``MongoClientSettings`` object.
+
+.. tip:: Builder Pattern
+
+   The builder pattern makes the settings immutable and supports 
+   a simpler construction through chaining.
+
+Example
+~~~~~~~
+
+This example demonstrates specifying a ``ConnectionString`` in a
+``MongoClientSettings``:
```

Comment:
suggestion: I think this can be simplified without impacting the meaning

```suggestion
This example demonstrates specifying a ``ConnectionString``:
```